css/v202209/symbiota/header.css

```diff
@@ -249,16 +312,28 @@ ul.menu {
   nav.top-menu {
     max-height: none;
     position: relative;
+    text-align: center;
+    padding-top: 0.75rem;
+    padding-bottom: 0.75rem;
+  }
+
+  ul.menu * {
+    vertical-align: middle;
   }
 
-  .menu li {
+  ul.menu > li {
```

Comment:
Since ul.menu is more specific than .menu, do we run the risk of losing styling on portals that are assigning the menu class to elements that are not ul?

Reply:
I am not aware of any portals where this would cause a problem.  In the current codebase I believe there is only one element that uses this class - so perhaps this should be converted to an id?